### PR TITLE
Add missing webhook enum values

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

**Review by d-wysocki — APPROVED** [review]

**Review by korycins — APPROVED** [review]

**Review by patrys — APPROVED** [review]

File: docs/developer/extending/apps.mdx [hunk 1]
```diff
@@ -95,6 +95,9 @@ Webhooks can be used to receive data from Saleor when particular events happen.
...
 | `PRODUCT_DELETED`           | A product is deleted.                                                                                                                                                                                                   |
+| `INVOICE_REQUESTED`         | An invoice for order requested.                                                                                                                                                                                             |
+| `INVOICE_DELETED`           | An invoice is deleted.                                                                                                                                                                                               |
+| `INVOICE_SENT`              | Invoice has been send.                                                                                                                                                                                               |
```
Comment (karolkielecki) [inline docs/developer/extending/apps.mdx]: An invoice is sent.

Comment (karolkielecki) [inline docs/developer/extending/apps.mdx]:
```suggestion
| `INVOICE_SENT`              | An invoice is sent.                                                                                                                                                                                          |
```

File: docs/developer/extending/apps.mdx [hunk 2]
```diff
@@ -95,6 +95,9 @@ Webhooks can be used to receive data from Saleor when particular events happen.
 | `PAGE_DELETED`              | A page is deleted.                                                                                                                                                                                                      |
 | `PRODUCT_UPDATED`           | A product is updated.                                                                                                                                                                                                   |
 | `PRODUCT_DELETED`           | A product is deleted.                                                                                                                                                                                                   |
+| `INVOICE_REQUESTED`         | An invoice for order requested.                                                                                                                                                                                             |
```
Comment (karolkielecki) [inline docs/developer/extending/apps.mdx]: An invoice for an order is requested.

Comment (karolkielecki) [inline docs/developer/extending/apps.mdx]:
```suggestion
| `INVOICE_REQUESTED`         | An invoice for an order is requested.                                                                                                                                                                                            |
```

File: docs/developer/extending/apps.mdx [hunk 3]
```diff
@@ -78,23 +78,26 @@ Webhooks can be used to receive data from Saleor when particular events happen.
...
-| `ORDER_CREATED`             | A new order is placed.                                                                                                                                                                                                  |
+| `FULFILLMENT_CREATED`       | A new fulfillment is created.                                                                                                                                                                                           |
+| `INVOICE_DELETED`           | An invoice is deleted.                                                                                                                                                                                               |
+| `INVOICE_REQUESTED`         | An invoice for order requested.                                                                                                                                                                                             |
```
Comment (krzysztofwolski) [inline docs/developer/extending/apps.mdx:87]:
```suggestion
| `INVOICE_REQUESTED`         | An invoice for an order is requested.                                                                                                                                                                                             |
```

File: docs/developer/extending/apps.mdx [hunk 4]
```diff
@@ -78,23 +78,26 @@ Webhooks can be used to receive data from Saleor when particular events happen.
...
+| `INVOICE_SENT`              | Invoice has been send.                                                                                                                                                                                               |
```
Comment (krzysztofwolski) [inline docs/developer/extending/apps.mdx:88]:
```suggestion
| `INVOICE_SENT`              | An invoice is sent.                                                                                                                                                                                               |
```

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**